rebar.config

```diff
@@ -46,7 +46,7 @@
         ex_doc
     ]},
     {ci, [
-        {do, "default as test check"},
+        check,
```

Comment:
CI wasn't failing when using the test profile, but by using the default, it detected the `xref` errors in `arizona_parse_transform`.

Reply:
Why not avoid the module, simply? I'm not really sure of the context here, though.

Reply:
See https://github.com/arizona-framework/arizona/pull/334/files#r2206140796